front/app/containers/Admin/projects/project/admin_phase_email_wrapper/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import { useParams } from 'react-router-dom';
+
+import useCampaigns from 'api/campaigns/useCampaigns';
+import useSupportedCampaignNames from 'api/campaigns/useSupportedCampaignNames';
+
+import useLocalize from 'hooks/useLocalize';
+
+import CampaignRow from 'containers/Admin/messaging/AutomatedEmails/CampaignRow';
+import { stringifyCampaignFields } from 'containers/Admin/messaging/AutomatedEmails/utils';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from './messages';
+
+const AdminPhaseEmailWrapper = () => {
+  const localize = useLocalize();
+  const { phaseId } = useParams();
+  const supportedCampaignNames =
+    useSupportedCampaignNames({ phaseId }).data?.data.attributes || [];
+  const contextCampaigns = useCampaigns({
```

Comment:
This is a little bit unconventional in the FE :thinking: Typically we'd have this format when fetching data:

`
const { data: supportedCampaigns } = useSupportedCampaignNames({ phaseId });
const supportedCampaignNames = supportedCampaigns?.data.attributes || [];
`

Could we use something like that for the supportedCampaignNames and globalCampaigns? :)

Reply:
Sure, making the change. Is there any reason for doing it this way?

Reply:
Maybe since we often want to get multiple attributes from the `data` being returned (though I know in this case we just want the data.attributes)? :thinking: